zigup.zig

```diff
@@ -292,11 +292,7 @@ pub fn main2() !u8 {
             defer allocator.free(install_dir);
             const compiler_dir = try std.fs.path.join(allocator, &[_][]const u8{ install_dir, version_string });
             defer allocator.free(compiler_dir);
-            if (std.mem.eql(u8, version_string, "master")) {
-                @panic("set default to master not implemented");
-            } else {
-                try setDefaultCompiler(allocator, compiler_dir, .verify_existence);
-            }
+            try setDefaultCompiler(allocator, compiler_dir, .verify_existence);
```

Comment:
I'm not sure if this is correct or not.  Could you add a test for this (in `test.zig`)?

Reply:
I have been using it since the day I created this PR, it is working though. Not sure about any hidden edge cases, but I did tested it a bit then and no problem found, afterall master is effectively a symlink to something.

Anyways, I will try to add a test once I figure out how it works.

Reply:
I tried to add a test case here: https://github.com/marler8997/zigup/pull/34/commits/ab2413082b023f11529b1c1d1da944b31416f49b 

Couldn't figure out a better way since there's nothing to check in the output